src/elements/content-explorer/ContentExplorer.js

```diff
@@ -411,6 +443,9 @@ class ContentExplorer extends Component<Props, State> {
         } else {
             this.setState({ rootName });
         }
+        if (viewMode === VIEW_MODE_GRID && items) {
```

Comment:
This logic is used multiple times, so I thought about making a `shouldFetchThumbnails` function, but flow would complain about `items` potentially being undefined below.  Would a `shouldFetchThumbnails` function be worth it?